package.json

```diff
@@ -43,6 +43,7 @@
     "@angular-devkit/build-optimizer": "0.0.13",
     "@angular-devkit/schematics": "0.0.17",
     "@schematics/angular": "0.0.27",
+    "@nrwl/nx": "https://github.com/nrwl/nx-build",
```

Comment:
Note: I'd be okay (with devrel approval) to give you publish rights to `@schematics/bazel` if that works with you.